test-case/check-pause-suspend-resume-release.sh

```diff
@@ -172,13 +171,14 @@ expect {
 }
 AUDIO
 
-ret=$?
-#flush the output
-echo
-if [ $ret -ne 0 ]; then
-    sof-process-kill.sh
-    [[ $? -ne 0 ]] && dlogw "Kill process catch error"
-    exit $ret
-fi
-sof-kernel-log-check.sh "$KERNEL_CHECKPOINT"
-exit $?
+    ret=$?
```

Comment:
About the rename, because the original name is check-pause-release-suspend-resume.sh, it does not represent the test that our case will be done, so I changed it to check-pause-suspend-resume-release.sh. I will raise a PR to change the case name.